src/hooks/transactions/useParseSignedTransactions.tsx

```diff
@@ -43,6 +44,7 @@ export function useParseSignedTransactions(
         }
 
         if (signedTransactions.length > 0) {
+          // TODO: check if the transactions are same as the ones in the redux store
```

Comment:
Missing comment or the TODO is not implemented yet?

Reply:
This will be resolved in a future PR